### DISCUSSION TITLE
Connection UI Error

### DISCUSSION THREAD

**Discussion by umair7228** [apache/airflow#43422] [General]
### Apache Airflow version

2.10.2

### What happened?

Airflow is running correctly but creating a connection It shows

Something bad has happened. For security reasons detailed information about the error is not logged.

  * You should check your webserver logs and retrieve details of this error.

  * When you get the logs, it might explain the reasons, you should also Look for similar issues using:

     * [GitHub Discussions](https://github.com/apache/airflow/discussions)
     * [GitHub Issues](https://github.com/apache/airflow/issues)
     * [Stack Overflow](https://stackoverflow.com/questions/tagged/airflow)
     * the usual search engine you use on a daily basis

    All those resources might help you to find a solution to your problem.

  * if you run Airflow on a Managed Service, consider opening an issue using the service support channels

  * only after you tried it all, and have difficulty with diagnosing and fixing the problem yourself,
    get the logs with errors, describe results of your investigation so far, and consider creating a
    [bug report](https://github.com/apache/airflow/issues/new/choose) including this information.

Python version: 3.12.3
Airflow version: 2.10.2
Node: redacted
-------------------------------------------------------------------------------
Error! Please contact server admin.

### How to reproduce

I don't understand the problem It is just shows the error you may see

### Operating System

Windows

### Deployment

Other

### Deployment details

I'm running airflow in EC2 Instance

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! If you are willing to raise PR to address this issue please do so, no need to wait for approval.

**Comment by tirkarthi** [comment]
Can you please post the traceback if any from the websever logs removing sensitive information?

**Comment by potiuk** [comment]
Converting it to a discussion - withoug the details, this is at most a discussion and up to the submitter (@umair7228 )  to provide enough information to classify it differently.

Also there are few things you might do before @umair7228. You mentioned Windows (and window is clearly not supported for Airflow - if you read "prerequisites" in our docs you will find out relevant information why and link to issues where you can contribute Windows support.  Secondly - you might look at your errors and look for similar issues. 

There is a WTForms new release issue - for example - that migh hit the users who do not follow our recommendation in the "installation" page of ours including constraints (which you can search in our docs) which is likely to produce similiar behaviour. But this is a guess only.

Do your own fact-checking first by looking at your particular erorr and comparing it with orher issues.

This will also help to save precious time of volunteers here who spend their evenings and weekends on trying to respond to such issuees, so "doing your own check" and "provide all information" is a good idea if you consider you get the software for free and if you care a little about other's people personal time.

**Reply by umair7228** [reply]
It has solved, Thank You and I was run airflow on Linux through AWS EC2 Instance.

**Reply by slalanne** [reply]
hey @umair7228, I'm having a similar issue and you indicated that you've solved this, are you able to tell me what the issue was and the steps you took to solve it?
I'd greatly appreciate it

**Reply by umair7228** [reply]
Run this command 'pip show wtforms' it will shows the version of wtforms 3 or above then uninstall wtforms using 'pip uninstall wtforms'
and reinstall 2.3.3 version using command 'pip install wtforms==2.3.3', it will resolve the issue.
